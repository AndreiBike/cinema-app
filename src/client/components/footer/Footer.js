import React from 'react';
import styles from './Footer.module.css'

const Footer = (props) => {
  return (
    <div className={styles.footer}>
      <span id = {styles.footerText}> {props.footerText} </span>
    </div>
  )
}

export default Footer;