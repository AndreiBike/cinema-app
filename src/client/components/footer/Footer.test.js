import React from 'react';
import {configure, render} from 'enzyme';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';
import Footer from './Footer';

configure({adapter: new Adapter()});

const footerLabelsTest = {
  netflixRoulette: ["NETFLIX", "ROULETTE"],
};

const component = renderer.create(<Footer footerLabels = {footerLabelsTest} />).toJSON();

describe("<Footer> Snapshot", () => {
  it('+++capturing snapshot of Footer', () => {
    expect(component).toMatchSnapshot();
  })
});


