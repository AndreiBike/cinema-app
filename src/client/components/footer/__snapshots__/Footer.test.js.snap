// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer> Snapshot +++capturing snapshot of Footer 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "footer",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "class": "footer-bold-text",
        },
        "children": Array [
          Object {
            "data": "NETFLIX",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "text",
          },
        ],
        "name": "span",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "data": "ROULETTE",
          "next": null,
          "parent": [Circular],
          "prev": [Circular],
          "type": "text",
        },
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
      Object {
        "data": "ROULETTE",
        "next": null,
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {
            "class": "footer-bold-text",
          },
          "children": Array [
            Object {
              "data": "NETFLIX",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
          ],
          "name": "span",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": null,
          "type": "tag",
          "x-attribsNamespace": Object {
            "class": undefined,
          },
          "x-attribsPrefix": Object {
            "class": undefined,
          },
        },
        "type": "text",
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;

exports[`Footer Snapshot +++capturing snapshot of Footer 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "footer",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "class": "footer-bold-text",
        },
        "children": Array [
          Object {
            "data": "NETFLIX",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "text",
          },
        ],
        "name": "span",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "data": "ROULETTE",
          "next": null,
          "parent": [Circular],
          "prev": [Circular],
          "type": "text",
        },
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
      Object {
        "data": "ROULETTE",
        "next": null,
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {
            "class": "footer-bold-text",
          },
          "children": Array [
            Object {
              "data": "NETFLIX",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
          ],
          "name": "span",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": null,
          "type": "tag",
          "x-attribsNamespace": Object {
            "class": undefined,
          },
          "x-attribsPrefix": Object {
            "class": undefined,
          },
        },
        "type": "text",
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
